### ISSUE TITLE
Bug: 21-4140 Focus Unnecessarily set on the Intro page

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4941]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Focus is manually set to the h1 on the forms landing page (https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/). This is okay within the flow of an app or form, but on the landing/intro  page, focus should be untouched.

If focus is changed automatically without a user action, confusion can be caused, especially for people using screen readers or keyboard navigation. This is okay within the form flow when going to the next page.

## What I expected to happen

I expected focus to be unmanaged on the forms introduction page. This means we would not see a focus ring on the h1 or anywhere on the page, on page load

## Reproducing

**URL(s) where this bug occurs**

- https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/
-
-

**Steps to reproduce:**

1. Load or navigate to https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/
2. See the focus ring
3.
4.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [X] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other
